src/common/devtools.js

```diff
@@ -148,7 +148,7 @@ const report =
 
 
 export const init =
-  ({isActive}/*:{isActive:boolean}*/)/*:[Model, Effects<Action>]*/ => {
+  ({isActive}/*:{isActive:boolean}*/):[Model, Effects<Action>] => {
```

Comment:
Argument type here also seems to be commented, likely because transformation script did not considered annotations with anonymous types.